Formula/neko.rb

```diff
@@ -24,6 +24,16 @@ class Neko < Formula
   uses_from_macos "sqlite"
   uses_from_macos "zlib"
 
+  on_linux do
+    depends_on "apr"
+    depends_on "apr-util"
+    depends_on "httpd"
+    # On mac, neko uses carbon. On Linux it uses gtk2
+    depends_on "gtk+"
+    depends_on "pango"
+    depends_on "sqlite"
```

Comment:
I think this should be `uses_from_macos "sqlite"`

Reply:
Actually `depend_on "sqlite"` wasn't even needed, I missed that `uses_from_macos "sqlite"` was already in the formula.